qbft/spectest/tests/controller_spectest.go

```diff
@@ -25,6 +26,7 @@ type RunInstanceData struct {
 	InputValue           []byte
 	InputMessages        []*qbft.SignedMessage
 	ControllerPostRoot   string
+	ControllerPostState  types.Root `json:"-"` // Field is ignored by encoding/json
```

Comment:
so do we need both `ControllerPostRoot` and `ControllerPostState`?

Reply:
since it looks like we're using `ControllerPostState` as Root

Reply:
its a good question
ideally we want to have the ControllerPostState only and generate the tests.json with this state (since we can get the root from this state), the issue is that we need to adjust the way ssv-node is working with those json files.

Reply:
Currently we don't serialize the `PostState` json
When we change this, we can discuss the question again

As Oleg said, this is not for this PR